packages/graphql-fixtures/src/fill.ts

```diff
@@ -264,7 +264,10 @@ function fillType(
       ? array.map((value: any) =>
           fillType(
             unwrappedType.ofType,
-            field,
+            {
+              ...field,
+              responseName: (field.responseName += parentFields.indexOf(field)),
```

Comment:
This is only actually appending a `1`. If the general approach is correct here, I'll generate something unique here.

Reply:
you should just be able to use the index from the map here

Reply:
I don't think you want to mutate the `responseName` though, that might have other consequences (for example, the filler for a field gets the details about the field, including its response name, and is probably not expecting it to differ from the original value. I think we may need another argument for this.

Reply:
can we not just add a new `fieldIndex` to `FieldDetails` (defaulting to `undefined`) and refactor how we build the key to consider the existence of this field index?